internal/pkg/term/selector/selector.go

```diff
@@ -534,9 +545,91 @@ func (s *CFTaskSelector) Task(msg, help string, opts ...GetDeployedTaskOpts) (st
 	return choice, nil
 }
 
+// DeployedJob has the user select a deployed job. Callers can provide either a particular environment,
+//// a particular service to filter on, or both.
+func (s *DeploySelector) DeployedJob(msg, help string, app string, opts ...GetDeployedWorkloadOpts) (*DeployedJob, error) {
```

Comment:
It looks like this method has a lot of duplication as `DeployedService`. I wonder if we should instead add a `deployedWorkloads` and having both `DeployedServices` and `DeployedJobs` to use it.